packages/react-instantsearch/src/connectors/connectMenu.js

```diff
@@ -28,7 +28,7 @@ function getValue(name, props, searchState) {
   return name === currentRefinement ? '' : name;
 }
 
-const sortBy = ['count:desc', 'name:asc'];
+const sortBy = ['isRefined', 'count:desc', 'name:asc'];
```

Comment:
I think `isRefined` should be the default anytime we have a list of items because we don't know the number of expected items and the widget is not aware if the visibility of selected items is offered by another widget.